client/app/vps/modal/shortcut/kvm/novnc/index.less

```diff
@@ -0,0 +1,6 @@
+/* stylelint-disable-next-line selector-type-no-unknown */
+dedicated-vps-kvm-novnc {
+  #noVNC_screen {
+    height: 600px;
```

Comment:
No media queries ? This works on mobile phones ?

Reply:
Yes it works.

Reply:
🖤